eventbridge.tf

```diff
@@ -2,16 +2,18 @@ data "aws_region" "current" {}
 data "aws_caller_identity" "current" {}
 
 locals {
-  appspec_content = "{\"version\": 1,\"Resources\": [{\"TargetService\": {\"Type\": \"AWS::ECS::Service\",\"Properties\": {\"TaskDefinition\": \"${aws_ecs_task_definition.default[0].arn}\",\"LoadBalancerInfo\": {\"ContainerName\": \"${var.ecs_load_balancers[0].container_name}\",\"ContainerPort\": ${var.ecs_load_balancers[0].container_port}}}}]}"
-  appspec_sha256 = sha256(local.appspec_content)
+  container_name  = length(var.ecs_load_balancers) > 0 ? var.ecs_load_balancers[0].container_name : "rift"
+  container_port  = length(var.ecs_load_balancers) > 0 ? var.ecs_load_balancers[0].container_port : "80"
+  appspec_content = "{\"version\": 1,\"Resources\": [{\"TargetService\": {\"Type\": \"AWS::ECS::Service\",\"Properties\": {\"TaskDefinition\": \"${aws_ecs_task_definition.default[0].arn}\",\"LoadBalancerInfo\": {\"ContainerName\": \"${local.container_name}\",\"ContainerPort\": ${local.container_port}}}}]}"
+  appspec_sha256  = sha256(local.appspec_content)
 }
 
 ## IAM
 
 resource "aws_iam_role" "event_bridge_codedeploy" {
-  count     = var.deployment_controller_type == "CODE_DEPLOY" ? 1 : 0
+  count = var.deployment_controller_type == "CODE_DEPLOY" ? 1 : 0
 
-  name = "EventBridgeCodeDeploy-${var.ecs_load_balancers[0].container_name}"
+  name = "EventBridgeCodeDeploy"
```

Comment:
pretty sure these names have to be unique. I don't think there's anywhere we'd be using it twice per account at the moment, but it's not hard to imagine.